src/lib/containers/CardContainerLogic.tsx

```diff
@@ -8,18 +8,30 @@ import { getNextPageOfData } from '../utils/modules/queryUtils'
 import { GenericCardSchema, IconOptions } from './GenericCard'
 import { insertConditionsFromSearchParams, KeyValue, SQLOperator } from '../utils/modules/sqlFunctions'
 
-export type InternalLinkConfiguration = {
+export interface TitleLinkConfig {
   baseURL: string
-  columnValues: string []
+  URLColumnNames: string []
 }
 
+export type LabelLink = {
```

Comment:
CardLink {
baseURL: string
  // the keys that will go into the url
  URLColumnNames: string []
}

LabelLink extends CardLink